pkg/yurthub/proxy/remote/remote.go

```diff
@@ -101,6 +103,21 @@ func (rp *RemoteProxy) modifyResponse(resp *http.Response) error {
 				h.Add("Transfer-Encoding", "chunked")
 			}
 		}
+
+		// 404 Not Found: The CRD may have been unregistered and should be updated locally as well
+		if resp.StatusCode == http.StatusNotFound {
+			gvr := schema.GroupVersionResource{
+				Group:    info.APIGroup,
+				Version:  info.APIVersion,
+				Resource: info.Resource,
+			}
+			gvk, err := serializer.YurtHubRESTMapperManager.CRDRESTMapper.KindFor(gvr)
```

Comment:
please try do not reference `serializer.YurtHubRESTMapperManager` out of `serializer`

Reply:
Thanks for reminding. I combined RESTMapperManager with CacheManager, and the interface will be exposed by CacheManager.